ecs/init.sls

```diff
@@ -0,0 +1,32 @@
+{% from "ecs/map.jinja" import ecs with context -%}
+
+{% for dir in ecs_agent_dirs %}
+{{ dir }}:
+  file.directory
+
+{%- endfor %}
+
+
+net.ipv4.conf.all.route_localnet:
+    sysctl.present:
+      - value: 1
+
+iptables-dnat:
+  iptables.append:
+    - table: nat
+    - rule: "-p tcp -d 169.254.170.2 --dport 80 -j DNAT --to-destination 127.0.0.1:51679"
```

Comment:
Trying to understand what this rule does.
IP is AWS metadata service, and we send that to local?